Objects/fileobject.c

```diff
@@ -33,7 +34,7 @@ PyFile_FromFd(int fd, const char *name, const char *mode, int buffering, const c
     PyObject *io, *stream;
     _Py_IDENTIFIER(open);
 
-    io = PyImport_ImportModule("io");
+    io = PyImport_ImportModule("_io");
```

Comment:
Can't you use io.open here? If there is a reason for using _io directly, maybe add a comment to explain why.

Reply:
It must be ``_io`` because import of ``io`` module may cause a cascade of additional auditing events.

Reply:
Yep, we may need this function to open `io.py` in order to import it.